components/script/dom/dedicatedworkerglobalscope.rs

```diff
@@ -358,6 +359,7 @@ impl DedicatedWorkerGlobalScope {
     #[allow(unsafe_code)]
     pub fn forward_error_to_worker_object(&self, error_info: ErrorInfo) {
         let worker = self.worker.borrow().as_ref().unwrap().clone();
+        let pipeline_id = worker.clone().root().global().pipeline_id();
```

Comment:
Instead we can use `self.upcast::<GlobalScope>().pipeline_id()` and that will avoid the panic.